src/Xamarin.Android.NamingCustomAttributes/Java.Interop/ExportAttribute.cs

```diff
@@ -0,0 +1,30 @@
+using System;
+
+namespace Java.Interop {
+
+	[Serializable]
+	[AttributeUsage (AttributeTargets.Method | AttributeTargets.Constructor, 
+			AllowMultiple=false, 
+			Inherited=false)]
+#if !JCW_ONLY_TYPE_NAMES
+	public
+#endif  // !JCW_ONLY_TYPE_NAMES
+	partial class ExportAttribute : Attribute {
```

Comment:
Removing those copies break build. Since it should be rather dealt by those who work on those registration types (which I guess is likely *you*) and I have no idea about the *ideal* state of them, I don't plan to deal with them. You can make different changes once they are merged.